omero/sysadmins/index.rst

```diff
@@ -43,9 +43,37 @@ series of commands.
     unix/server-install-homebrew
     unix/server-binary-repository
     unix/server-postgresql
-    unix/install-web
     advanced-install
 
+********************
+OMERO.web Deployment
+********************
+
+This section provides guidance on how to install and set up OMERO.web on
+any of the supported UNIX and UNIX-like platforms. Specific walkthroughs are
+provided for several systems, with detailed step-by-step instructions.
+OMERO.web can be either deployed with the OMERO.server or **separately** from 
+the OMERO.server. Deploying separately is **recommended**.
```

Comment:
I think a very general introductory sentence could be useful here: "OMERO.web is a python client of OMERO that provides a web-based UI and JSON API." Otherwise newbies might assume that OMERO.web is a "lite" or web-only version of OMERO.